charts/tezos-signer-forwarder/templates/statefulset.yaml

```
@@ -0,0 +1,101 @@
+{{- range .Values.signers }}
+{{- $name := .name }}
+{{- $readiness_probe_path := .readiness_probe_path }}
+{{- $signer_port := .signer_port }}
+{{- $signer_metrics := .signer_metrics }}
+{{- $signers := . }}
+{{- range $i := until (len .endpoints) }}
+{{- $endpoint := index $signers.endpoints $i }}
```

I believe this can just be:
```suggestion
{{- range $i, $endpoint := .endpoints }}
```